posthog/models/feature_flag.py

```diff
@@ -754,7 +754,8 @@ def can_user_edit_feature_flag(request, feature_flag):
             final_level = org_level
         else:
             final_level = role_level
-
+        if OrganizationResourceAccess.objects.filter(organization=request.user.organization).exists() is False:
```

Comment:
Isn't this the same codepath as Line 749 above? Where org_level defaults to `CAN_ALWAYS_EDIT` ?

Why do we need this special casing here?

Reply:
I can clean this up afterwards

Reply:
This most likely occurred after I enabled role access for one of our customers who did set feature flags to "view only" access

Reply:
> When we filter with objects directly on the model I don't think it does the organization filtering for us

Hmm, if this is true, then you should add this condition to Line 745 I think, as otherwise we'll be prone to more bugs 😅 .

Reply:
And sure, get it in first :)

Reply:
>otherwise we'll be prone to more bugs 😅 .
But not today, because there are no other resource models except feature flags, so it's fine to clean up afterwards if you don't want to wait again for tests to run